src/components/molecules/button/index.js

```diff
@@ -2,24 +2,40 @@ import { LitElement, html, css, unsafeCSS } from 'lit-element';
 import { classMap } from 'lit-html/directives/class-map';
 import buttonCSS from './index.scss';
 
+const icon = () => html`
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    class="m-button__icon"
+    width="12"
+    height="7"
+    viewBox="0 0 12 7"
+  >
+    <g fill-rule="evenodd" stroke-linecap="round" stroke-width="2">
+      <path d="M.5 3.5H11M9 7l2.798-3.5L9 0" />
+    </g>
+  </svg>
+`;
+
 class AXAButton extends LitElement {
   static tagName = 'axa-button';
-  static styles = css`${unsafeCSS(buttonCSS)}`;
+  static styles = css`
+    ${unsafeCSS(buttonCSS)}
+  `;
 
   static get properties() {
     return {
-      // red, white
-      color: { type: String },
-      // sm, lg
-      size: { type: String },
       // button, submit, reset
       type: { type: String },
+      icon: { type: String },
 
-      ghost: { type: Boolean },
+      secondary: { type: Boolean },
+      large: { type: Boolean },
+      inverted: { type: Boolean },
+      cta: { type: Boolean },
       motion: { type: Boolean },
       disabled: { type: Boolean },
 
-      onClick: { type: Function },
+      onClick: { type: String },
```

Comment:
on line 48 we check for type function but here is string.. is this intended as not finished code or a n error?